<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetServices</class>
 <widget class="QWidget" name="widgetServices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Services</string>
  </property>
  <property name="windowIcon">
   <iconset resource="services.qrc">
    <normaloff>:/icons/services48.png</normaloff>:/icons/services48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Services</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listServices"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="services.qrc"/>
 </resources>
 <connections/>
</ui>
